api/.env.example

```diff
@@ -22,6 +22,7 @@ FILES_ACCESS_TIMEOUT=300
 
 # celery configuration
 CELERY_BROKER_URL=redis://:difyai123456@localhost:6379/1
+CELERY_GLOBAL_PREFIX={celery_global_prefix}
```

Comment:
CELERY_GLOBAL_PREFIX should be left blank by default to avoid affecting existing users